src/vision/room_luminance/Readme.md

```diff
@@ -0,0 +1,32 @@
+﻿# Simple Visual System Needs
+
+This repository contains the code for detecting room luminance in general. Currently, it includes the following features:
+
+* General Room Luminance Detection and Classification.
+* Hand/Object Block Recognition
+
+Moreover, the repository will have the following features by the end of development.
+* Room Silence
+* Room Occupation
+* and other simple visual system needs
```

Comment:
You should probably list all of those that you can support, and those you think you might be able to, soon.  For example, can you detect a camera flash? If so, the robot could say something like "are you taking photos of me?" or "I like posing for photos".

Attention: @audreyleeann and @demaris and @anyrobot once this gets merged, you'll be able to use this info in the chatbot dialogs. I think this could lead to some slick demos (urm, I guess "slick" has both positive and negative implications...)